Formula/gtk+3.rb

```diff
@@ -111,3 +113,27 @@ def post_install
     system "./test"
   end
 end
+
+__END__
+diff --git a/gdk/quartz/gdkwindow-quartz.c b/gdk/quartz/gdkwindow-quartz.c
+index f868f35..db2e58f 100644
+--- a/gdk/quartz/gdkwindow-quartz.c
++++ b/gdk/quartz/gdkwindow-quartz.c
+@@ -2478,7 +2478,7 @@ gdk_quartz_window_set_functions (GdkWindow    *window,
+   gboolean min, max, close;
+ 
+   g_return_if_fail (GDK_IS_WINDOW (window));
+-
++#if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_12
```

Comment:
It will compile on El Capitan as long as Xcode 8 is installed. There is no guarantee that people who compile their packages from source on El Cap have actually done that.

Interestingly, Apple has removed the `setStyleMask` method from the 10.12 API, in favour of `@property styleMask`, which itself was changed in 10.12 from `NSUInteger` to `NSWindowStyleMask` (an NSUInteger enum).